### PR TITLE
Spanish Translation of Readme

### PR DESCRIPTION
**Pull request by ucdstudent95618**
I have translated the English Readme.md to Spanish (Readme.es.md).

### CODE REVIEW

**Comment by sinancepel** [comment]
Miguel,

This looks great; thank you for contributing!

File: README.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ See [INSTALL.md](https://github.com/facebook/pyre-check/blob/master/INSTALL.md)
 
 See [CONTRIBUTING.md](https://github.com/facebook/pyre-check/blob/master/CONTRIBUTING.md) for how to help out.
 
+## Languages
```
Comment (sinancepel) [inline README.md]: One small detail: Could you change the format of linking to mimic the structure in https://github.com/tiimgreen/github-cheat-sheet/blob/master/README.md, and have [Español] as the link name?

Comment (ucdstudent95618) [reply README.md]: Thank you  :)

Comment (ucdstudent95618) [reply README.md]: I will update it.

**Comment by sinancepel** [comment]
Landing and closing :)

**Pull request closed without merging**